lib/taskana-test-api/src/main/java/pro/taskana/testapi/extensions/TestContainerExtension.java

```diff
@@ -73,7 +78,12 @@ private static void copyValue(String key, Store source, Store destination) {
 
   private static String determineSchemaName() {
     String uniqueId = "A" + UUID.randomUUID().toString().replace("-", "_");
-    return EXECUTION_DATABASE == DB.POSTGRES ? uniqueId.toLowerCase() : uniqueId;
+    if (EXECUTION_DATABASE == DB.ORACLE) {
+      uniqueId = uniqueId.substring(0, 20);
```

Comment:
```
    String uniqueId = "A" + UUID.randomUUID().toString().replace("-", "");
    if (EXECUTION_DATABASE == DB.ORACLE) {
      uniqueId = uniqueId.substring(0, 25);
```
I might be paranoid, but I don't want to risk one execution where for some reason the same schema name is created and therefore something breaks. Therefore I like to max this out as much as possbile ;)

Reply:
```
uniqueId = uniqueId.substring(0, 26);
```